src/main/resources/application.yaml

```diff
@@ -75,16 +75,24 @@ service-config:
     - service: probate
       jurisdiction: PROBATE
       transformation-url: ${TRANSFORMATION_URL_PROBATE}
-      case-type-ids: []
+      case-type-ids:
+        - GrantOfRepresentation
+        - Caveat
+        - StandingSearch
+        - WillLodgement
```

Comment:
what are `case-type-ids` and what does an empty list mean

Reply:
There is a comment on the whole configuration section. I've updated it. Not introducing anything in this PR, just concrete values.